src/csi/util.rs

```diff
@@ -190,14 +199,15 @@ pub fn spawn_grpc_task<R: Send + 'static>(
     sink: UnarySink<R>,
     task: impl Future<Output = DatenLordResult<R>> + Send + 'static,
 ) {
-    smol::spawn(async move {
+    // spawn a thread to prevent the current thread from being blocked
+    // cannot use tokio spawn instead or there will be deadlock
```

Comment:
This comment is not accurate now. Please change to 
// grpcio library is running outside of tokio default runtime, so need a dedicated runtime for grpc task.

Reply:
done